charts/refinery/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: refinery
 description: Chart to deploy Honeycomb Refinery
 type: application
 version: 2.8.0
```

Comment:
```suggestion
version: 2.8.1
```

Reply:
I didn't see this bump in other bumping PRs. But OK!